plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/commands/edit/CopyFeaturesCommand.java

```diff
@@ -285,7 +288,7 @@ private Query createQuery( ILayer sourceLayer, Filter filter, Layer targetLayer,
 
         queryAttributes.putAll(FeatureUtils.createAttributeMapping(sourceSchema, targetSchema));
         Set<String> properties = new HashSet(queryAttributes.values());
-        return new DefaultQuery(sourceSchema.getName().getLocalPart(), filter, properties
+        return new Query(sourceSchema.getName().getLocalPart(), filter, properties
```

Comment:
It the switch from DefaultQuery to Query related to the bugfix or just a cleanup here?

Reply:
Just cleanup since DefaultQuery has been marked as deprecated

Reply:
Tested context menu in map panel with right mouse -> copy (of selected feature) and pased it on map ✔️ 
tested also to copy more than one feature ✔️ 
copy in map-panel and paste onto layer works as expected ✔️ 

Thanks again for fixing this issue!